assets/training/finetune_acft_hf_nlp/components/validation/common/spec.yaml

```diff
@@ -22,10 +22,38 @@ inputs:
     type: string
     optional: true
     default: serverless
-    description: compute to be used for model_eavaluation eg. provide 'FT-Cluster' if your
+    description: compute to be used for finetune component eg. provide 'FT-Cluster' if your
       compute is named 'FT-Cluster'. Special characters like \ and ' are invalid in the parameter value.
       If compute cluster name is provided, instance_type field will be ignored and the respective cluster will be used
 
+  compute_model_import:
+    type: string
+    optional: true
+    default: serverless
+    description: compute to be used for model import component eg. provide 'FT-Cluster' if your
+      compute is named 'FT-Cluster'. Special characters like \ and ' are invalid in the parameter value.
+      If compute cluster name is provided, instance_type field will be ignored and the respective cluster will be used
+
+  compute_data_preprocess:
+    type: string
+    optional: true
+    default: serverless
+    description: compute to be used for data preprocess component eg. provide 'FT-Cluster' if your
+      compute is named 'FT-Cluster'. Special characters like \ and ' are invalid in the parameter value.
+      If compute cluster name is provided, instance_type field will be ignored and the respective cluster will be used
+
+  task_name:
+    type: string
+    optional: true
+    default: SingleLabelClassification
+    description: Text Classification task type
```

Comment:
Can we update the description here to something like NLP task type?

Reply:
Updated to "Finetuning task type"